aggsender/types/certificate_build_params.go

```diff
@@ -23,8 +23,7 @@ type CertificateBuildParams struct {
 	RetryCount                     int
 	LastSentCertificate            *CertificateInfo
 	L1InfoTreeRootFromWhichToProve *treetypes.Root
-	AggchainProof                  []byte
-	CustomChainData                []byte
```

Comment:
is this removed?

Reply:
It is not removed, just moved to the new field:

```
AggchainProof *AggchainProof
```

Which is basically everything the `prover` returned on `GenerateAggchainProof` request.